### PR TITLE
Rhoadesj/dev

### PR DESCRIPTION
**Pull request by rhoadesScholar**

### CODE REVIEW

**Comment by mzouink** [comment]
first we need to pull the main into this branch and fix conflicts

File: dacapo/apply.py
Comment (rhoadesScholar) [inline dacapo/apply.py:1]: initial implementation of apply.py

File: dacapo/cli.py
Comment (rhoadesScholar) [inline dacapo/cli.py:1]: initial implementation of apply.py

File: dacapo/validate.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import torch
 
 from pathlib import Path
+from reloading import reloading
```
Comment (mzouink) [inline dacapo/validate.py:15]: reload is a debug thing, caused many problems in my runs. somehow it doesn't detect the location of dacapo library

File: dacapo/utils/voi.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 import scipy.sparse as sparse
 
 
+# TODO: Why are we not using this?
```
Comment (mzouink) [inline dacapo/utils/voi.py:13]: you mean for loss calculation or for plotting ?

File: dacapo/utils/balance_weights.py
Comment (mzouink) [inline dacapo/utils/balance_weights.py:1]: this should be a gull pull request, let's start with an issue about weight balancing and drive weight balancing things from there

File: dacapo/store/local_weights_store.py
Comment (rhoadesScholar) [inline dacapo/store/local_weights_store.py:1]: get rid of

**Comment by mzouink** [comment]
Still exist only the non merged parts

File: dacapo/experiments/datasplits/datasets/arrays/zarr_array.py
Comment (rhoadesScholar) [inline dacapo/experiments/datasplits/datasets/arrays/zarr_array.py:1]: dismiss for now

File: dacapo/experiments/run.py
Comment (rhoadesScholar) [inline dacapo/experiments/run.py:1]: now in dev/start

File: dacapo/experiments/starts/start.py
Comment (rhoadesScholar) [inline dacapo/experiments/starts/start.py:1]: now in dev/start

File: dacapo/experiments/tasks/evaluators/evaluator.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/evaluators/evaluator.py:1]: now in dev/validation

File: dacapo/experiments/tasks/evaluators/instance_evaluation_scores.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/evaluators/instance_evaluation_scores.py:1]: now in dev/validation

File: dacapo/experiments/tasks/evaluators/instance_evaluator.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/evaluators/instance_evaluator.py:1]: now in dev/validation

File: dacapo/experiments/trainers/gp_augments/__init__.py
Comment (rhoadesScholar) [inline dacapo/experiments/trainers/gp_augments/__init__.py:1]: now in dev/augmentations

File: dacapo/experiments/trainers/gp_augments/gaussian_noise_config.py
Comment (rhoadesScholar) [inline dacapo/experiments/trainers/gp_augments/gaussian_noise_config.py:1]: now in dev/augmentations

File: dacapo/experiments/trainers/gp_augments/simple_config.py
Comment (rhoadesScholar) [inline dacapo/experiments/trainers/gp_augments/simple_config.py:1]: now in dev/augmentations

File: dacapo/experiments/validation_scores.py
Comment (rhoadesScholar) [inline dacapo/experiments/validation_scores.py:1]: now in dev/augmentations

File: dacapo/gp/elastic_augment_fuse.py
Comment (rhoadesScholar) [inline dacapo/gp/elastic_augment_fuse.py:1]: now in dev/augmentations

File: dacapo/predict.py
Comment (rhoadesScholar) [inline dacapo/predict.py:1]: remove this change

File: dacapo/store/local_array_store.py
Comment (rhoadesScholar) [inline dacapo/store/local_array_store.py:1]: remove comment

File: dacapo/utils/voi.py (hunk 2 above)
Comment (rhoadesScholar) [reply dacapo/utils/voi.py:13]: this is a dumb comment and should be removed. lol

File: dacapo/validate.py
Comment (rhoadesScholar) [inline dacapo/validate.py:1]: now in dev/validation

File: dacapo/gp/dacapo_create_target.py
Comment (rhoadesScholar) [inline dacapo/gp/dacapo_create_target.py:1]: dismiss comment until bug happens ;)

File: dacapo/experiments/tasks/hot_distance_task.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/hot_distance_task.py:1]: now in rhoadesj/hot_distance

File: dacapo/experiments/tasks/hot_distance_task_config.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/hot_distance_task_config.py:1]: now in rhoadesj/hot_distance

File: dacapo/experiments/tasks/losses/__init__.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/losses/__init__.py:1]: now in rhoadesj/hot_distance

File: dacapo/experiments/tasks/losses/hot_distance_loss.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/losses/hot_distance_loss.py:1]: now in rhoadesj/hot_distance

File: dacapo/experiments/tasks/predictors/hot_distance_predictor.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/predictors/hot_distance_predictor.py:1]: now in rhoadesj/hot_distance

File: dacapo/validate.py (hunk 1 above)
Comment (rhoadesScholar) [reply dacapo/validate.py:15]: removed :)

File: dacapo/experiments/tasks/post_processors/dummy_post_processor.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/post_processors/dummy_post_processor.py:1]: added to dev/post-process

File: dacapo/experiments/tasks/post_processors/post_processor.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/post_processors/post_processor.py:1]: added to dev/post-process

File: dacapo/experiments/tasks/post_processors/threshold_post_processor.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/post_processors/threshold_post_processor.py:1]: added to dev/post-process

File: dacapo/experiments/tasks/post_processors/watershed_post_processor.py
Comment (rhoadesScholar) [inline dacapo/experiments/tasks/post_processors/watershed_post_processor.py:1]: added to dev/post-process

File: dacapo/experiments/trainers/gunpowder_trainer.py
Comment (rhoadesScholar) [inline dacapo/experiments/trainers/gunpowder_trainer.py:1]: added to dev/trainers

File: dacapo/experiments/trainers/gunpowder_trainer_config.py
Comment (rhoadesScholar) [inline dacapo/experiments/trainers/gunpowder_trainer_config.py:1]: added to dev/trainers

File: dacapo/utils/balance_weights.py
Comment (rhoadesScholar) [inline dacapo/utils/balance_weights.py:1]: added to dev/trainers

**Review by rhoadesScholar — COMMENTED** [review]
All changes have been split to separate branches for merging. This branch is now derelict.
